src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -26,11 +29,17 @@ type WorldsDashboardProps = { userId: UserId };
 export const WorldsDashboard: React.FC<WorldsDashboardProps> = ({ userId }) => {
   const { ownedVenues } = useOwnedVenues({ userId });
   const worlds = useWorlds();
+  const { ownWorlds } = useOwnWorlds();
+
+  const { isAdminUser: isMasterAdmin } = useAdminRole({ userId });
```

Comment:
Not sure about the name. `superadmin` (or SA for short) is usually referenced elsewhere in issues.
Might as well bite the bullet and rename to `isSuperAdmin` for all uses of the hook.

```suggestion
  const { isSuperAdmin } = useAdminRole({ userId });
```